init.php

```diff
@@ -18,6 +18,7 @@
 // HttpClient
 require __DIR__ . '/lib/HttpClient/ClientInterface.php';
 require __DIR__ . '/lib/HttpClient/CurlClient.php';
+require __DIR__ . '/lib/HttpClient/StreamingClientInterface.php';
```

Comment:
I think the order here might matter? You seem to be getting a build break:

```
PHP Fatal error:  Uncaught Error: Interface 'Stripe\HttpClient\StreamingClientInterface' not found in /home/runner/work/stripe-php/stripe-php/lib/HttpClient/CurlClient.php:27
```

As `CurlClient` needs `StreamingClientInterface`, you need to swap these.

Reply:
Yes! Thank you!!